src/aws-api-mcp-server/awslabs/aws_api_mcp_server/core/common/config.py

```diff
@@ -105,6 +105,7 @@ def get_user_agent_extra() -> str:
 TRANSPORT = get_transport_from_env()
 HOST = os.getenv('AWS_API_MCP_HOST', '127.0.0.1')
 PORT = int(os.getenv('AWS_API_MCP_PORT', 8000))
+STATELESS_HTTP = get_env_bool('AWS_API_MCP_STATELESS_HTTP', False)
```

Comment:
Have we confirmed with AppSec that this is acceptable and doesn't require any review? I remember part of why streamable http was not a big concern is because we rely on session id. With stateless-http we seem to be loosing that session protection?